internal/cmd/operator-sdk/generate/bundle/bundle.go

```diff
@@ -343,3 +352,26 @@ func (c bundleCmd) findManagerEnvironment(col *collector.Manifests) ([]corev1.En
 		"could not find manager deployment, should have label %s=%s", managerLabel, managerLabelValue,
 	)
 }
+
+// pinImages is used to replace all image tags in the given manifests with digests
```

Comment:
WDYT about we add some test to ensure that works as should be?
WDYT about we just use the sample generated to generate a new sample such as v3/memcached-operator-digest to ensure that it works? 

Could we have here samples with some specific options as we have in Kubebuilder to do the checks?
Would that be a good idea?

Reply:
@camilamacedo86 This PR was already approved and merged [here][https://github.com/operator-framework/operator-sdk/pull/5523], but for some reason when we merged the feature branch into master, it was missing this commit. Adding automated testing is its own separate issue due to some other issues we found.